src/sickle.ts

```diff
@@ -55,8 +56,17 @@ class Annotator {
 
   private emit(str: string) { this.output.push(str); }
 
+  private logWithIndent(message: string) {
+    var prefix = '';
+    for (let i = 1; i < this.indent; i++) {
+      prefix += '| ';
+    }
+    console.log(prefix + message);
+  }
+
   private visit(node: ts.Node) {
-    // console.log('node:', (<any>ts).SyntaxKind[node.kind]);
+    this.indent++;
```

Comment:
nit/optional: maybe add a sanity check `indent == 0` when done visiting nodes? That way we can be certain we don't mess this up.